app/assets/stylesheets/spotlight/typeahead.css

```diff
@@ -37,7 +37,7 @@
 }
 
 .twitter-typeahead {
-  z-index: 1000 !important; /* get typeahead to float on top of e.g. jcrop */
```

Comment:
If we've done away with jcrop, I wonder if we can remove this override? I think it happens on e.g. the appearance page when you're cropping a masthead and try to select a different item from the repository.